magenta/music/musicxml_parser_test.py

```diff
@@ -0,0 +1,190 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test to ensure correct import of MusicXML."""
+
+from collections import defaultdict
+import os.path
+
+# internal imports
+import tensorflow as tf
+from magenta.music.musicxml_parser import MusicXMLDocument
+from magenta.music.musicxml_reader import musicxml_to_sequence_proto
+
+class MusicXMLParserTest(tf.test.TestCase):
```

Comment:
I think it would be good to verify the things described below. For example:
- flute_scale should have an F-major scale of 8 quarter notes. You could verify that the notes and durations are parsed correctly.
- clarinet_scale should definitely verify that the note values are parsed and transposed correctly.
- band_score could verify a few chords, time signature changes, tempo changes, etc.

Our MIDI test cases are admittedly sparse. For a better example, take a look at our QuantizedSequence tests, which construct a NoteSequence and an expected QuantizedSequence that should result: https://github.com/tensorflow/magenta/blob/master/magenta/music/sequences_lib_test.py#L128